ci-operator/step-registry/single-node/conf/azure/single-node-conf-azure-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SINGLE_NODE_AZURE_INSTANCE_TYPE="Standard_D32s_v3"
```

Comment:
I expect there to be one set of folks maintaining these single-node conf step(s), so it might we worth a single `single-node/conf` command instead of the platform-specific sub-steps.  On the other hand, I'm not in the maintainer set, so doesn't matter for me ;).